README.md

```diff
@@ -20,3 +21,108 @@ This module uses the Admin API for updating Kong workspace config.
 ``` bash
 node app.js
 ```
+
+## Workspace YAML syntax
+
+Workspaces are stored in a YAML file (default: workspaces.yaml)
```

Comment:
@ashosborne this part will be changed after templating. the conf yaml will be generated on the fly after the compilation of the template with data file. Admins of the platform will have to store the workspace configuration in a json data file and pass it as env variable.